surrealdb/clients/ws.py

```diff
@@ -0,0 +1,284 @@
+"""
+Copyright © SurrealDB Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import asyncio
+from types import TracebackType
+from typing import Any
+from typing import Awaitable
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Type
+
+from aiohttp import ClientSession
+from aiohttp import ClientWebSocketResponse
+from aiohttp import WSMsgType
+
+from ..common import json as jsonlib
+from ..common.exceptions import SurrealWebsocketException
+from ..common.id import generate_id
+from ..models import RPCRequest
+from ..models import RPCResponse
+
+__all__ = ("SurrealDBWSClient",)
+
+
+class SurrealDBWSClient:
```

Comment:
Again, I don't like this naming with `SurrealDB` in the class name. Perhaps 
```suggestion
class WebsocketClient:
```
Better yet, you could make a `ws/` dir and a `client/` dir to allow for similiar class names

Reply:
i think removing `SurrealDB` removes any clarity it is actually for Surreal. imagine in a large codebase you are importing multiple clients for multiple usecases. `WebsocketClient` and `HTTPClient` does not really provide much clarity on it's usecase... i'd rather keep `SurrealDB` in the name in some form

Reply:
> i think removing `SurrealDB` removes any clarity it is actually for Surreal. imagine in a large codebase you are importing multiple clients for multiple usecases. `WebsocketClient` and `HTTPClient` does not really provide much clarity on it's usecase... i'd rather keep `SurrealDB` in the name in some form

The user can always `import surrealdb` and just use `surrealdb.WebsocketClient`. Not sure if its really necessary to name a class within a package the package name. However, if you think its really desirable then your call

Reply:
probably true, i might aswell rename